parm/fv3lam_cmaq_post_avblflds.xml

```diff
@@ -0,0 +1,6147 @@
+<?xml version="1.0"?>
```

Comment:
This file should removed and all new variables for AQ are added in post_avblflds.xml.

Reply:
The additional variables in this additional file and the other extra file (*cmaq*) as in the new makefile are consistent with the  "*lam*".  They are used for not put dummy variables in other domains.

Reply:
The UPP should have an unified avblflds.xml. Please remove all new AQF definitions into post_avblflds.xml.

Reply:
removed